docs/infrastructure/microk8s-quickstart.md

```diff
@@ -41,70 +41,93 @@ Install MicroK8s on the machine:
 3.  Check that MicroK8s is running:
 
     ```bash
-    $ microk8s.status --wait-ready
-    microk8s is running
-    addons:
-    knative: disabled
-    jaeger: disabled
-    fluentd: disabled
-    gpu: disabled
-    cilium: disabled
-    storage: disabled
-    registry: disabled
-    rbac: disabled
-    ingress: disabled
-    dns: disabled
-    metrics-server: disabled
-    linkerd: disabled
-    prometheus: disabled
-    istio: disabled
-    dashboard: disabled
+    microk8s.status --wait-ready
     ```
 
 ## 3. Configuring MicroK8s
 
-Now that MicroK8s is running on the machine we can proceed to enabling the necessary plugins and installing the Helm client and server:
+Now that MicroK8s is running on the machine we can proceed to enabling the necessary addons and installing the Helm client and server:
 
-1.  Enable the following plugins on MicroK8s:
+1.  Configure MicroK8s to allow privileged containers:
 
     ```bash
     sudo mkdir -p /var/snap/microk8s/current/args
     sudo echo "--allow-privileged=true" >> /var/snap/microk8s/current/args/kube-apiserver
     microk8s.status --wait-ready
+    ```
+
+2.  Enable the following MicroK8s addons:
```

Comment:
@paulopontesm, what do you think about having the `microk8s.status --wait-ready` after enabling each MicroK8s addon? Hopefully this will help avoid the race condition that caused some addons to not be enabled correctly.

Reply:
I just realized I'm reverting a change that @andreaTP made in https://github.com/codacy/chart/pull/340/commits/cbf8bf709ba696d5f5f30393f6eb34e23fc08029#diff-eaa6927aebd91897b0dfeac25e036d13L71.

However, I think it's safer to include the "waiting time" between enabling each addon as I've seen the glitch happen both on my local Ubuntu virtual machine as in AWS EC2 instances.

Reply:
I think we should add it. 
If it helps with flakiness I believe it's ok. Also, it's not a shady command and should be quite straight forward for anyone that it following the guide to understand what it does

Reply:
Although, it looks like the --wait sometimes is asynchronous and it doesn't wait for the resources of the addon to be created, it just waits for the reply that the instruction was received successfully. DNS seems to be the one that generates more problems. Should we create a disclaimer or something saying that this could happen? Just re-run...

Reply:
Indeed it makes sense to add the disclaimer / note.

I was also thinking of moving the DNS addon to be activated last, but since [it is required by other addons](https://microk8s.io/docs/addon-dns) (probably the Ingress needs it?) I just left it as the first one.